docs/manuals/userman/versioning/index.md

```diff
@@ -34,6 +34,10 @@ a Taipy application.
   algorithms, investigating the impacts of a parameter change, etc. For more information, please refer to
   [experiment mode](experiment_mode.md).
 
+!!! warning "Available in Taipy Enterprise edition"
+
+    The production mode is relevant only to the Enterprise edition of Taipy.
+
```

Comment:
I would move this within the third bullet point, probably after it.

Reply:
it is now moved inside the bullet point